client/css/style.css

```diff
@@ -1041,6 +1041,7 @@ kbd {
 
 #chat .chat {
 	overflow: auto;
+	overflow-x: hidden;
```

Comment:
Is it worth adding a comment here about reasoning?

Reply:
If I'm not wrong, this CSS line already existed before, but got removed when I refactored the flexbox stuff.